Formula/fstar.rb

```diff
@@ -36,32 +35,39 @@ def install
                                            "$(DATE_EXEC) '+%Y-%m-%dT%H:%M:%S%z'"
 
     resource("z3").stage do
-      system "python", "scripts/mk_make.py", "--prefix=#{libexec}"
-      system "make", "-C", "build"
-      system "make", "-C", "build", "install"
+      # F* warns if the Z3 git hash doesn't match
+      system "python", "scripts/mk_make.py", "--prefix=#{libexec}", "--githash=1f29cebd4df6"
+      cd "build" do
+        system "make"
+        system "make", "install"
+      end
     end
 
     system "opam", "init", "--no-setup"
-    inreplace "opamroot/compilers/4.05.0/4.05.0/4.05.0.comp",
+    inreplace "opamroot/compilers/4.06.0/4.06.0/4.06.0.comp",
       '["./configure"', '["./configure" "-no-graph"' # Avoid X11
 
-    system "opam", "switch", "4.05.0"
+    system "opam", "switch", "4.06.0"
```

Comment:
current OCaml in brew is 4.06.1. That's no good?
```
depends_on "ocaml" => :build
```

Reply:
I haven't tried, but it should work.